src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -275,10 +277,17 @@ function ReimbursementAccountPage({reimbursementAccount, route, onfidoToken, pol
         BankAccounts.openReimbursementAccountPage(stepToOpen, subStep, ignoreLocalCurrentStep ? '' : localCurrentStep, policyID);
     }
 
+    function shouldClearReimbursementAccount() {
+        return backTo !== ROUTES.WORKSPACE_WORKFLOWS.getRoute(policyID);
+    }
+
```

Comment:
I'm not sure that this approach is correct. If we're clearing the key, does that mean it's a temporary draft key only used while we're adding a bank account? If so, I think we should use a different Onyx key that is permanent and doesn't get cleared.

Reply:
Yes, I think the new command should return all the data this page needs to load (if we're already not fetching it elsewhere in OpenApp for example)

Reply:
The OpenPolicyWorkflowsPage API is now in production